RAT/utils/plotting.py

```diff
@@ -17,9 +18,9 @@ def __init__(self, row: int = 1, col: int = 2):
 
         Parameters
         ----------
-        row : int
+        row : int, default: 1
               The number of rows in subplot
-        col : int
+        col : int, default: 2
```

Comment:
Should the default be 2 here for the general figure routine? Do you think it makes more sense to have the default as 1 here and set two columns in plot_ref_sld?

Reply:
I will set default to 1